Packs/AwakeSecurity/Integrations/AwakeSecurity/AwakeSecurity.py

```diff
@@ -98,7 +102,7 @@ def returnResults(contents, outerKey, innerKey, humanReadable, dbotScore, generi
     if genericContext:
         entryContext.update(genericContext)
 
-    demisto.results({
+    return ({
```

Comment:
do these parens do anything?
```suggestion
    return {
```